vega/base.py

```diff
@@ -51,6 +53,8 @@ def _ipython_display_(self):
             metadata={'jupyter-vega': '#{0}'.format(id)}
         )
 
-    def display(self):
+    def display(self, renderer=None):
         """Render the visualization."""
+        if renderer:
+            self.renderer = renderer
```

Comment:
I agree with that. 

But I was hoping for a way to quickly swap between the renderers without creating a new Vega object.

But ya, the side effect can be nasty.